clients/ui/frontend/src/app/pages/modelCatalog/components/ValidatedModelCard.tsx

```diff
@@ -0,0 +1,247 @@
+                  borderRadius: '50%',
+                }}
+              />
+              <Content component={ContentVariants.small} className={text.textColorStatusSuccess}>
+                Success
+              </Content>
+            </Flex>
+            <Link to={catalogModelDetailsFromModel(model.name, source?.id)}>
+              <Button
+                data-testid="model-catalog-detail-link"
+                variant="link"
+                tabIndex={-1}
+                isInline
+                style={{ fontSize: '1.25rem', fontWeight: 'bold' }}
+              >
+                {truncate ? (
+                  <span>{getModelName(model.name)}</span>
+                ) : (
+                  <span>{getModelName(model.name)}</span>
+                )}
+              </Button>
+            </Link>
+          </div>
+        </CardTitle>
+      </CardHeader>
+      <CardBody>
+        <div>
+          <div style={{ paddingLeft: '8px', marginBottom: '16px' }}>
+            <Flex
+              alignItems={{ default: 'alignItemsCenter' }}
+              justifyContent={{ default: 'justifyContentSpaceBetween' }}
+            >
+              <Flex
+                alignItems={{ default: 'alignItemsCenter' }}
+                gap={{ default: 'gapSm' }}
+                wrap="nowrap"
+              >
+                <ChartBarIcon />
+                <span
+                  data-testid="validated-model-accuracy"
+                  style={{ fontSize: '1.25rem', fontWeight: 'bold' }}
+                >
+                  {metrics.accuracy}%
+                </span>
+              </Flex>
+              <Flex
+                alignItems={{ default: 'alignItemsCenter' }}
+                gap={{ default: 'gapSm' }}
+                wrap="nowrap"
+              >
+                <span className="pf-v5-c-content pf-m-small">Average accuracy</span>
+                <Tooltip
+                  content="Average accuracy
+
+The weighted average of normalized scores from all benchmarks. Each benchmark is normalized to a 0-100 scale. All normalized benchmarks are then averaged together."
+                  position="top"
+                  className="pf-v6-c-tooltip"
+                >
+                  <HelpIcon style={{ cursor: 'help' }} />
+                </Tooltip>
+              </Flex>
+            </Flex>
+          </div>
+
+          {/* Hardware Configuration and Latency */}
+          <div style={{ paddingLeft: '8px', marginBottom: '16px' }}>
+            <Flex justifyContent={{ default: 'justifyContentSpaceBetween' }}>
+              <div>
+                <Content component={ContentVariants.small} style={{ color: '#6a6e73' }}>
+                  Hardware
+                </Content>
+                <Content component={ContentVariants.p} data-testid="validated-model-hardware">
+                  {metrics.hardwareCount}x{metrics.hardware}
+                </Content>
+              </div>
```

Comment:
A couple things here, comparing this to the mockups.
- The subtitles "Hardware", "RPS/rep." and "TTFT" should appear below the metrics, not above.
- I think "p" is the wrong component to use for the metric, it puts too much whitespace between the caption and the metric. Maybe just don't put that part in a Content component, it can go in a span.
- The metric numbers should be bold, using `className="pf-v6-u-font-weight-bold"`.
- Avoid inline styles, especially hard coded colors. The captions here can just be the default color they get from ContentVariants.small, it's already slightly lighter than regular text.